From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Haris Okanovic <haris.okanovic@ni.com>
Date: Fri, 18 Jul 2014 10:04:38 -0500
Subject: [PATCH] Excluding HAVE_WINSOCK2_H and HAVE_WS2TCPIP_H on PharLAP

Upstream-Status: Inappropriate [Pharlap specific]

Updated for curl 7.72, and in particular, also hiding the blanket ws2tcpip.h
include.

Signed-off-by: Richard Tollerton <rich.tollerton@ni.com>

diff --git a/lib/config-win32.h b/lib/config-win32.h
index 316043d81..04c25f432 100644
--- a/lib/config-win32.h
+++ b/lib/config-win32.h
@@ -144,6 +144,7 @@
 /* Define if you have the <winsock.h> header file. */
 #define HAVE_WINSOCK_H 1
 
+#ifndef PHARLAP
 /* Define if you have the <winsock2.h> header file. */
 #ifndef __SALFORDC__
 #define HAVE_WINSOCK2_H 1
@@ -153,7 +154,7 @@
 #ifndef __SALFORDC__
 #define HAVE_WS2TCPIP_H 1
 #endif
-
+#endif PHARLAP
 /* ---------------------------------------------------------------- */
 /*                        OTHER HEADER INFO                         */
 /* ---------------------------------------------------------------- */
@@ -720,7 +721,9 @@ Vista
 
 /* On MinGW the ADDRESS_FAMILY typedef was committed alongside LUP_SECURE,
    so we use it to check for the presence of the typedef. */
+#ifndef PHARLAP
 #include <ws2tcpip.h>
+#endif
 #if !defined(__MINGW32__) || defined(LUP_SECURE)
 /* Define to use Unix sockets. */
 #define USE_UNIX_SOCKETS
-- 
2.28.0

